### ISSUE TITLE
Appendix G ManifestationType

### ISSUE THREAD

**Issue by paulduchesne** [FIAF/fiaf-cataloguing-manual#30]
Appendix G ManifestationType 

Also double-check whether this is in correct place - should it be alongside Identifier in an Item record?

**Comment by natashafairbairn** [comment]
No, it shouldn't be there. Have edited and removed in the branch document for Issue 31, which deals with other corrections in the table in Appendix G - so correction will be picked up and applied in main draft from that one.

Natasha
